C-Plus-Plus/cp/README.md

```diff
@@ -1 +1,6 @@
-a
+Peak Element
```

Comment:
Revert this. You don't need to update this sub-folder README. You just need to add hyperlink of your program in the README file of C-Plus-Plus folder. Refer to a recently merged C++ PR to get a better idea of this.